codegen.ts

```diff
@@ -30,6 +30,14 @@ const config: CodegenConfig = {
           ActionItemCategory:
             "../models/ActionItemCategory#InterfaceActionItemCategory",
 
+          Advertisement: "../models/Advertisement#InterfaceAdvertisement",
+
+          CreateAdvertisementPayload:
+            "../models/Advertisement#InterfaceAdvertisement",
+
+          UpdateAdvertisementPayload:
+            "../models/Advertisement#InterfaceAdvertisement",
```

Comment:
Except the `Advertisement` mapping remove the other two, this configuration is only for mapping database model types to corresponding graphql types